addon/package.json

```diff
@@ -43,7 +43,8 @@
   "dependencies": {
```

Comment:
Should `ember-auto-import` be added as a dependency here? According to https://github.com/embroider-build/embroider/blob/main/packages/macros/README.md#hint it seems to be the case, but not sure.

Reply:
nope, ember-qunit is a v2 addon, so ember-auto-import isn't needed